community-operators/camel-k/camel-k.v0.3.3.clusterserviceversion.yaml

```diff
@@ -0,0 +1,321 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    categories: "Integration & Delivery"
+    repository: https://github.com/apache/camel-k
+    containerImage: docker.io/apache/camel-k:0.3.3
+    support: Camel
+    createdAt: 2019-05-23T15:12:00Z
+    certified: "false"
+    description: |-
+      Apache Camel K (a.k.a. Kamel) is a lightweight integration framework built from Apache Camel that runs natively on Kubernetes and is specifically designed for serverless and microservice architectures.
+    alm-examples: |-
+      [{
+        "apiVersion": "camel.apache.org/v1alpha1",
+        "kind": "IntegrationPlatform",
+        "metadata": {
+          "name": "example"
+        },
+        "spec": {
+          "resources": {
+            "contexts": [
+              "jvm"
+            ]
+          }
+        }
+      },
+      {
+        "apiVersion": "camel.apache.org/v1alpha1",
+        "kind": "Integration",
+        "metadata": {
+          "name": "example"
+        },
+        "spec": {
+          "source": {
+            "content": "// Add example Java code to create Integration",
+            "name": "Example.java"
+          }
+        }
+      },
+      {
+        "apiVersion": "camel.apache.org/v1alpha1",
+        "kind": "IntegrationContext",
+        "metadata": {
+          "name": "example"
+        }
+      },
+      {
+        "apiVersion": "camel.apache.org/v1alpha1",
+        "kind": "CamelCatalog",
+        "metadata": {
+          "name": "example"
+        }
+      },
+      {
+        "apiVersion": "camel.apache.org/v1alpha1",
+        "kind": "Build",
+        "metadata": {
+          "name": "example"
+        }
+      }]
+  name: camel-k-operator.v0.3.3
+  namespace: placeholder
+spec:
+  displayName: Camel K Operator
+  version: 0.3.3
```

Comment:
This misses a `replaces` property to point to the previous CSV>

Reply:
Thanks for the review. I've just added the missing field.